lib/Doctrine/DBAL/Platforms/SqlitePlatform.php

```diff
@@ -605,6 +614,11 @@ public function getInlineColumnCommentSQL($comment)
         return '--' . str_replace("\n", "\n--", $comment) . "\n";
     }
 
+    private function getInlineTableCommentSQL(string $comment) : string
```

Comment:
Is this method really needed?

Reply:
Still requiring feedback from @moufmouf

Reply:
The code to generate a SQL comment for a column or for a table is the same in `SqlitePlatform`.

I could directly call `$this->getInlineColumnCommentSQL($comment)` from here: 

https://github.com/doctrine/dbal/blob/4ea7c119d32d7fc1f7755cb5464a2f4f34e60fa4/lib/Doctrine/DBAL/Platforms/SqlitePlatform.php#L337-L342

But of course, we would use the "getInlineColumnCommentSQL" to retrieve a table comment and this seems weird. Creating an additional `getInlineTableCommentSQL` method makes it clear this is the behaviour that was intended.

I'm happy to remove the `getInlineTableCommentSQL`  method and directly point to `getInlineColumnCommentSQL` instead if you prefer. You tell me.